src/transactions/ChangeTrustTests.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 Stellar Development Foundation and contributors. Licensed
+// under the ISC License. See the COPYING file at the top-level directory of
+// this distribution or at http://opensource.org/licenses/ISC
+
+#include "main/Application.h"
+#include "overlay/LoopbackPeer.h"
+#include "util/make_unique.h"
+#include "main/test.h"
+#include "lib/catch.hpp"
+#include "util/Logging.h"
+#include "crypto/Base58.h"
+#include "lib/json/json.h"
+#include "TxTests.h"
+
+using namespace stellar;
+using namespace stellar::txtest;
+
+typedef std::unique_ptr<Application> appPtr;
+
+TEST_CASE("change trust", "[tx][setoptions]")
```

Comment:
the second string represents tags for this test.
Should probably be "[tx][changetrust]"